README

```diff
@@ -151,6 +151,29 @@ To create and play with a Test object from the example package,
 	}
 
 
+Parameters
+==========
+
+To pass extra parameters to the plugin, use a comma-separated
+parameter list separated from the output directory by a colon:
+
+
+	protoc --go_out=plugins=grpc,import_path=mypackage:. *.proto
+
+
+- `import_prefix=xxx` - a prefix that is prepended onto the beginning
```

Comment:
s/prepended/added/

Prefixes always appear at the start. Also Rob gets grumpy when people use "prepend".  ;-)